Sources/NIO/EventLoop.swift

```diff
@@ -725,6 +725,30 @@ public final class MultiThreadedEventLoopGroup: EventLoopGroup {
     private let shutdownLock: Lock = Lock()
     private var runState: RunState = .running
 
+    private static func runTheLoop(thread: NIOThread,
+                                   canEventLoopBeShutdownIndividually: Bool,
+                                   selectorFactory: @escaping () throws -> NIO.Selector<NIORegistration>,
+                                   initializer: @escaping ThreadInitializer,
+                                   _ callback: @escaping (SelectableEventLoop) -> Void) {
+        assert(NIOThread.current == thread)
+        initializer(thread)
+
+        do {
+            /* we try! this as this must work (just setting up kqueue/epoll) or else there's not much we can do here */
```

Comment:
Any reason to `try!` instead of relying on the `fatalError` below?

Reply:
none, fixed. thanks